src/components/Authentication/LoginRedirect.js

```diff
@@ -34,9 +58,7 @@ export class LoginRedirect extends Component {
     }
 
     render() {
-        return this.props.user
-            ? <Redirect to="/" />
-            : <CircularProgress />
+        return <div>if this is taking a long time please refresh<CircularProgress /></div>
```

Comment:
I think you took the wrong part of the rebase here

Reply:
Hmm I'm not sure why I put the text here but I did change the redirect logic - instead of relying on a `<Redirect/>` the component will route based on the logic in `useEffect` - through `props.history.push()`

Reply:
Ah okay, let's just take out the text and leave just the spinner